examples/hosts/app-integration/external-data/src/utilities/clientManager.ts

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Represents a Fluid client URL.
+ * This URL contains the client's Fluid session information necessary for broadcasting signals to.
+ */
+export type ClientSessionUrl = string;
+
+/**
+ * Represents the external data servers query url or uuid.
+ * This is the URL or the id of the external resource that the customer service needs to subscribe for at the external service.
+ */
+export type ExternalTaskListId = string;
```

Comment:
Honestly, I'm less than 100% convinced with these, in great part because they're part of the public interface of `ClientManager` so they have to be exported. But I'm more than 50% convinced that they help to make the class self-documenting, particularly for what would otherwise be a couple of `Map<string, Set(string)>`. The docs need a few fixes, though.

Reply:
Okay, I'm turning them into non-exports but keeping the types here!

Reply:
Oh, this compiles? Nice, I expected that it'd want them exported because they are part of the public interface. Maybe that's not the case for type aliases. I do wonder what Intellisense says about the methods that use them, when we are going to call those methods from outside the class.